anathema-widget-core/src/generator/testing.rs

```diff
@@ -151,6 +149,6 @@ impl<'e> TestNodes<'e> {
 }
 
 pub(crate) fn register_test_widget() {
-    Factory::register("test", TestWidgetFactory);
-    Factory::register("list", TestListWidgetFactory);
+    Factory::register("test", TestWidgetFactory).unwrap();
+    Factory::register("list", TestListWidgetFactory).unwrap();
```

Comment:
```suggestion
    Factory::register("test", TestWidgetFactory);
    Factory::register("list", TestListWidgetFactory);
```
This can't unwrap here, it would break tests